tests/zopen_check_install

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+#
+# Basic 'zopen install' check
+#
+
+WORKDIR="$1"
+INCLUDEDIR="$2"
+zopen_tool_binary=`whence zopen`
+zopen_tool_directory=$( dirname ${zopen_tool_binary} )
+
+# Display banner
+${INCLUDEDIR}/banner.sh "tests/check_install"
+
+
+echo "Tests starting ..."
+echo "-- Current Settings: ---"
+echo "WORKDIR: ${WORKDIR}"
+echo "zopen_tool_binary: ${zopen_tool_binary}"
+echo "zopen_tool_directory: ${zopen_tool_directory}"
+
+if ! zopen install zotsample -y; then
+    echo "Test of simple install failed."
+    return 4
+fi
+
+result=$(zopen install zotsampleport)
+expected_result="Usage error: please install using base name without port suffix.
+  try: zopen install zotsample"
+if [[ "${result}" != "${expected_result}" ]]; then
+    echo "Test of identifying port suffix failed."
+    return 4
```

Comment:
It may be better to integrate this test into zopen_check_basic_install: https://github.com/ZOSOpenTools/metaport/blob/main/tests/zopen_check_basic_install#L10C1-L11C1.
It also sets up a new zopen environment in the beginning.